runtime/util/vmargs.c

```diff
@@ -51,9 +51,9 @@
 #if defined(J9ZOS390)
 #include "atoe.h"
 #endif
-#if defined(RS6000) || defined(LINUX)
+#if defined(RS6000) || defined(LINUX) || defined(OSX)
 #define J9UNIX
-#endif
+#endif /* defined(RS6000) || defined(LINUX) || defined(OSX) */
```

Comment:
@gacholio `RS6000` actual means `AIX`, right?  If so, is there a more explicit macro for `AIX`?

Reply:
Probably lots - I think AIXPPC is defined. Personally, I don't want irrelevant changes cluttering this PR - if we want to fix those, do it separately.